### ISSUE TITLE
OBS Websocket v5 support

### ISSUE THREAD

**Issue by reedog117** [t2t2/obs-tablet-remote#131]
The websocket protocol has changed for v5 and the remote can no longer connect to OBS instances running this. In addition this version will be bundled by default in OBS 28.

**Comment by V3ntus** [comment]
Bump. Any progress here?

**Comment by paul-dab** [comment]
Bump, unable to use this since websocket 5 / OBS v28

**Comment by Rosuav** [comment]
Two solutions:
1) Get the OBS WebSocket 4.x-compat release, which can still be added as a third-party OBS 28 plugin. Be aware that this has some performance implications (the v4 protocol has a lot more costs to it, which are unavoidable).
2) Switch to a different client. I've been developing http://vol.rosuav.com/ to do the jobs that I was using t2t2 for.

Unfortunately, I don't think this tablet remote is getting any dev time.

**Comment by V3ntus** [comment]
Thanks @Rosuav

**Comment by PaulTGG** [comment]
@Rosuav Your site looks awesome!  Do you have the repo on github?  (And is it here?: https://github.com/Rosuav/obs-remote-volume)

**Comment by Rosuav** [comment]
Yep that's the one! Based on the analysis in my previous comment, I decided that it needed to grow features to do everything that I''d previously used t2t2 for, so now it does. The code is VERY messy because I wanted to retain protocol v4 compatibility, but some day I'll probably rip that out and go v5-only.

**Comment by ask-compu** [comment]
> Two solutions:
> 
> 1. Get the OBS WebSocket 4.x-compat release, which can still be added as a third-party OBS 28 plugin. Be aware that this has some performance implications (the v4 protocol has a lot more costs to it, which are unavoidable).
> 2. Switch to a different client. I've been developing http://vol.rosuav.com/ to do the jobs that I was using t2t2 for.
> 
> Unfortunately, I don't think this tablet remote is getting any dev time.

thanks! only thing i'd ask for on ur site is a dark mode

**Comment by Rosuav** [comment]
I'll let a dark-mode advocate worry about that.

**Comment by DigibitDK** [comment]
+1 on this

**Comment by LesterWolfCode** [comment]
i have an idea. Why don't you add a preview window. If it is possible.

**Comment by reedog117** [comment]
Considering there's been no movement and we're already up to OBS 29, I think it's safe to say this repo (and thus this remote) is likely not getting updated anytime soon. I've switched to @Rosuav 's http://vol.rosuav.com

**Comment by Rosuav** [comment]
Agreed. And if there are issues with my one, they can best be discussed on its own repository: https://github.com/Rosuav/obs-remote-volume
